app/controllers/videos_controller.rb

```diff
@@ -0,0 +1,41 @@
+class VideosController < ApplicationController
+  def index
+    videos = Video.all.order(:id)
+    render json: videos.as_json(only: [:id, :title, :release_date, :available_inventory]),
+           status: :ok
+  end
+  
+  def show
+    video = Video.find_by(id: params[:id])
+
+    if video.nil?
+      render json: {
+        errors: ["Not Found"],
+      }, status: :not_found
+
+      return
+    end
+
+    render json: video.as_json(only: [:title, :overview, :release_date, :total_inventory, :available_inventory]), status: :ok
+  end
+
+  def create
+    @video = Video.new(video_params)
+
+    if @video.save
+      render json: @video.as_json(only: [:id]), status: :created
+      return
+    else
+      render json: {
+        errors: @video.errors.messages,
+      }, status: :bad_request
```

Comment:
Here it _is_ appropriate to respond with a `:bad_request` status because a failure to save is most likely due to failed validations.
Technically, it's also possible that the save fails due to a connection failure, even if the validations pass, but that's more depth of error-handling than we've gotten into in our curriculum.